scripts/devportal/update-themes-and-modules.sh

```diff
@@ -1,21 +1,26 @@
 #!/bin/bash
-# update-px-modules.sh
-
-# UPDATE ALL PRONOVIX MODULES
+# update-themes-and-modules.sh
+# UPDATE ALL PRONOVIX MODULES AND THEMES.
 
 # ----- CONSTANTS ------
 declare -r MODULES_DIR='web/modules/devportal'
 declare -r THEMES_DIR='web/themes/devportal'
 declare -a modules
 declare -a themes
+declare -r DEVPORTAL_REMOTE_REPOSITORY_URL=${DEVPORTAL_REMOTE_REPOSITORY_URL-:"git@bitbucket.org:pronovix"}
 
-for dir in ./${MODULES_DIR}/*/; do
-    modules+=("$(basename "${dir}")")
-done
+if [[ -d ${MODULES_DIR} ]]; then
+  for dir in $(find ${MODULES_DIR} -mindepth 1 -maxdepth 1 -type d); do
```

Comment:
I had some issues with running the original script on Linux this is the reason why I changed.